### ISSUE TITLE
rkvm panics when starting the server on linux

### ISSUE THREAD

**Issue by colemickens** [htrefil/rkvm#7]
```
[INFO  server] Listening on 0.0.0.0:50258
thread 'main' panicked at 'there is no timer running, must be called from the context of Tokio runtime', /build/rkvm-unstable-ec404c69b38f7feff5103f898612734ee8d7ee95-vendor.tar.gz/tokio-0.3.6/src/time/driver/handle.rs:50:18
stack backtrace:
   0: rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::option::expect_failed
   3: tokio::time::driver::handle::Handle::current
   4: tokio::time::driver::sleep::Sleep::new_timeout
   5: server::run::{{closure}}
   6: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
   7: tokio::runtime::enter::Enter::block_on
   8: tokio::runtime::thread_pool::ThreadPool::block_on
   9: tokio::runtime::Runtime::block_on
  10: server::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

This is what I get if I try to start the server with a config file that looks very similar to the example one in this repo.

**Comment by htrefil** [comment]
Thank you for the issue. This was caused by having multiple incompatible tokio versions across dependencies and is now fixed by 4c02402. 

Also please when packaging rkvm, use one of the versions released here on GitHub, I don't want to gurantee that cloning the master branch will always work flawlessly.

**Comment by hpfr** [comment]
> Also please when packaging rkvm, use one of the versions released here on GitHub, I don't want to guarantee that cloning the master branch will always work flawlessly.

Can you cut a new release with the fixes to the futures dependency and the tokio dependency?

https://github.com/htrefil/rkvm/compare/0.2.0...master
